### PR TITLE
@gnilekaw: [WIP] Pass all files into mocha command

### PR DESCRIPTION
**Pull request by craigspaeth**
Using `for file in $@` to run `mocha` on each file slows the test suite down quite a bit because it can't leverage `require` caching. This changes the scripts/mocha.sh file to pass multiple files directly into mocha to leverage that again.

### CODE REVIEW

**Comment by craigspaeth** [comment]
Of course, this doesn't isolate tests from shared state bugs 😒  I'll mark this WIP and see if I can fix some of that.

File: scripts/mocha.sh [hunk 1]
```diff
@@ -2,11 +2,8 @@
...
-      --require source-map-support/register \
-      -r should \
-      --compilers coffee:coffee-script/register,js:babel-core/register \
-      -t 30000 $file
-  done
+mocha \
+  --require source-map-support/register \
+  -r should \
+  --compilers coffee:coffee-script/register,js:babel-core/register \
+  -t 30000 $@
```
Comment (gnilekaw) [inline scripts/mocha.sh]: Good call.

Comment (gnilekaw) [inline scripts/mocha.sh]: Did some quick and dirty benchmarks. This provides more than a 10% increase on the two files I tried.

**Comment by craigspaeth** [comment]
I'll reopen when I get a passing local build.

**Pull request closed without merging**